CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing to `rising`
+
+If you are interested in contributing to `rising`, you will either
+
+* implement a new feature
+
+or 
+
+* fix a bug.
+
+For both types of contribution, the process is roughly the same:
+
+1. File an issue at [this repo] and discuss 
+the issue with us! Maybe we can give you some hints towards 
+implementation/fixing.
+
+2. If you're not part of the core development team, we need you to create your own fork of [this repo], implement it there and create a PR to [this repo] afterwards.
+
+3. Start a new branch (in your fork if necessary) for the implementation of your issue. 
+Make sure to include basic unittests.
+
+> **Note:** To improve readability and maintainability, [PEP8 Style](https://www.python.org/dev/peps/pep-0008/) should always be followed (no exceptions).
+
+> **Note:** We prefer absolute imports over realtive ones..
```

Comment:
All imports inside the package should be absolute and every file should have an `__all__` section
All functions should be typed
Try to avoid spaghetti code -> kepp your functions short and precise and functions should only do what their name suggest
Choose meaningful names for your variables and classes